lib/rules/no-broken-super-chain.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * @fileOverview Prevent the absence of this._super() in init() calls or the use of this prior to this._super()
+ * @author Quinn C Hoyer
+ */
+
+const MESSAGES = {
+  noSuper: '\'this._super(...arguments)\' must be called in init in this Ember modules',
```

Comment:
I think the "in this Ember modules" part is extraneous. I think "'this._super(...arguments)' must be called in init" is enough.